tests/tekton-resources/tasks/teardown/awscli-eks.yaml

```diff
@@ -88,7 +88,38 @@ spec:
       aws cloudformation delete-stack --region $(params.region) --stack-name $(params.cluster-name)
       # Wait for the stack to be deleted
       aws cloudformation wait stack-delete-complete --region $(params.region) --stack-name $(params.cluster-name)
-      echo "Stack deleted successfully!"
+      if [ $? -ne 0 ]; then
+        # Get vpc id from stack that is blocking deletion
+        VPC_ID=$(aws cloudformation --region $(params.region) describe-stacks --stack-name $(params.cluster-name) | jq -r '.[].[].Outputs.[] | select(.OutputKey == "VpcId") | .OutputValue')
+        echo "$VPC_ID"
+        # Get security group ids from vpc that is blocking deletion. Have to get the security group this way because it doesn't report as part
+        # of the stack. Filter only security groups with cluster-name key to prevent deleting default sg
```

Comment:
Just a clarifying statement on this. This is because a test workflow is creating this SG in the CP VPC which our existing deleting logic fails to catch.